spec/acceptance/pe_status_check_spec.rb

```diff
@@ -302,6 +302,14 @@ class {'pe_status_check':
         expect(result.stdout).to match(%r{false})
         apply_manifest(absent)
       end
+
+      it 'if S0038 conditions for false are met' do
+        run_shell('mkdir /etc/puppetlabs/code/environments/pe_status_check_test_envs{1..100}')
```

Comment:
i think this isnt working in ubuntu which is why the test is failing

Reply:
stand corrected 
mkdir /etc/puppetlabs/code/environments/pe_status_check_test_envs{1..100}
does work in ubuntu i just checked

Reply:
I'll change this to use the API like Adrian suggested then see if the test works better at that point!